Spec/Auth.swift

```diff
@@ -1903,6 +1903,32 @@ class Auth : QuickSpec {
                         done()
                     }
                 }
+                
```

Comment:
Ok, well let's remove then and if this is a concern perhaps add as a separate test such as `localClockInSync` which simply fails with "Local clock is out of sync with Ably so tests may have unpredictable results"